lang/cplusplus/cplusplus.talon

```diff
@@ -0,0 +1,146 @@
+mode: command
+and mode: user.cplusplus
+mode: command
+and mode: user.auto_lang
+and code.language: cplusplus
```

Comment:
We've migrated to just using a tag for these matches as part of #776. Please update when you have a moment.

see https://github.com/knausj85/knausj_talon/blob/2d9f4afd73c2566fd5d3bc4a89251c746245cdbe/lang/csharp/csharp.py#L5

and

https://github.com/knausj85/knausj_talon/blob/2d9f4afd73c2566fd5d3bc4a89251c746245cdbe/lang/csharp/csharp.talon#L1

Reply:
Rebased + updated for the new tags.